src/main/scala/com/microsoft/hyperspace/index/IndexLogEntry.scala

```diff
@@ -111,6 +102,43 @@ object Content {
       None
     }
   }
+
+  /**
+   * Apply `func` to each file in directory recursively.
+   *
+   * @param prefixPath Root prefix
+   * @param directory Root directory
+   * @param func Function which would apply to current prefix and file
+   * @tparam T
+   * @return Result list of applying function to all files
+   */
+  def recFilesApply[T](
+                        prefixPath: Path,
```

Comment:
Could you try this if you are using intellij?
Setting => editor => coding style => scala
- formatter -> scalafmt
- configuration -> `dev\.scalafmt.conf`

ctrl + alt + l